src/main/resources/templates/studentSection.html

```diff
@@ -157,9 +138,6 @@ <h1>New Student</h1>
                     <div th:if="${blank}!= null" class="error">
                         <p th:text="${blank}"></p>
                     </div>
```

Comment:
I think in case of invalid password you won't have a message rendered under the corresponding field